github/util_v4_branch_protection.go

```diff
@@ -366,3 +404,71 @@ func getBranchProtectionID(repoID githubv4.ID, pattern string, meta interface{})
 
 	return nil, fmt.Errorf("Could not find a branch protection rule with the pattern '%s'.", pattern)
 }
+
+func getActorIds(data []string, meta interface{}) ([]string, error) {
+	var actors []string
+	for _, v := range data {
+		id, err := getNodeIDv4(v, meta)
+		if err != nil {
+			return []string{}, err
+		}
+		log.Printf("[DEBUG] Retrieved node ID for user/team : %s - node ID : %s", v, id)
+		actors = append(actors, id)
+	}
+
+	return actors, nil
+}
+
+// Given a string that is either a username or team slug, return the
+// node id of the user or team it is referring to. Team slugs must be provided
+// with the organization name as prefix (Ex.: exampleorg/exampleteam). Usernames
+// must be provided with the "/" prefix otherwise getNodeIDv4 assumes that
+// the provided string is a node ID.
```

Comment:
Is this specification from the GitHub API/documented anywhere public?

Reply:
I was thinking more of the `/` prefix vs. infix behavior. Regardless, I think it's intuitive enough.